pages/linux/nixos-rebuild.md

```diff
@@ -30,3 +30,7 @@
 - Build the configuration and open it in a virtual machine:
 
 `sudo nixos-rebuild build-vm`
+
+- List available generations similar to the boot loader menu:
+
+`nixos-rebuild list-generations --json`
```

Comment:
@Managor Is it clear from this example that `--json` should primarily be omitted?

`ls`, for example, has `ls -1` suggesting to use the entire command including the option. Without the option the command would not do what the description says:

![image](https://github.com/user-attachments/assets/cdaf789d-8934-4304-b180-c7137de03152)

Reply:
We try not to do optionality. Make a command and then describe what the command does. If something is optional, we usually tell the user with words.

Reply:
Understand. I removed the `--json` option. Users will probably use `--help` to look up whether some other form of output is available.